MapboxVision/src/main/java/com/mapbox/vision/utils/observable/Observable.kt

```diff
@@ -0,0 +1,19 @@
+package com.mapbox.vision.utils.observable
+
+import com.mapbox.vision.mobile.core.utils.delegate.DelegateWeakRef
+
+interface Observable<T> {
```

Comment:
Suggest to rename to `CompositeObserver` or `MulticastObserver`

Reply:
Also is it worth to have specific interface for that? Can we use just open class with the necessary methods, `CompositeListener`?

Reply:
discussed